src/components/PublicProfile.vue

```diff
@@ -65,18 +135,45 @@
           source: traitsRef,
           asObject: true,
         },
-        user: {
+        viewedUser: {
           source: schoolsRef.child(this.school).child(this.userId),
           asObject: true,
         },
       }
     },
+    methods: {
+      saveNewReview: function(){
+        let review = {
+          comment: this.additional_comments,
+          metrics: {
+            cleanliness: this.cleanliness,
+            loudness: this.loudness,
+            respectfulness: this.respectfulness,
+            sociability: this.sociability,
+          },
+          rater: this.user.displayName,
```

Comment:
It looks like you forgot to update this 'this.user' reference to 'this.viewedUser'

Reply:
That is alright, it is referring to the user in AuthMixin